src/components/Database/Database.scss

```diff
@@ -0,0 +1,16 @@
+.Database {
+  display: flex;
+  background-color: #fff;
```

Comment:
nit: should we be ordering our css a-z?

Reply:
Sure, I've re-ordered them. In general, I don't think we need to be strict about this, especially because we don't actually have that many properties per rule.